tests/Feature/Invoices/InvoiceAmountModelTest.php

```
@@ -103,7 +103,7 @@ public function it_calculates_invoice_totals_with_payments(): void
             'payment_date'   => '2024-01-10',
         ]);
 
-        $service = app(InvoiceAmountService::class);
+        $model = app(InvoiceAmountService::class);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$model'. (undefined)

(UnusedLocalVariable)